util/src/main/java/tc/oc/pgm/util/chunk/ChunkVector.java

```diff
@@ -20,20 +21,24 @@ public static ChunkVector of(Chunk chunk) {
     return new ChunkVector(chunk.getX(), chunk.getZ());
   }
 
-  public static ChunkVector ofBlock(int x, int y, int z) {
+  public static ChunkVector ofChunkVector(int x, int z) {
+    return new ChunkVector(x, z);
+  }
+
+  public static ChunkVector ofBlock(int x, int z) {
```

Comment:
If it's `ofBlock`, a block has X, Y and Z. Even if the chunk doesn't use the Z for anything, it's still part of the block, and this function is taking "a block" as input parameter, it's impl detail that the Y is discarded
```suggestion
  public static ChunkVector ofBlock(int x, int y, int z) {
```